content/en/agent/faq/ec2_imdsv2_transition_payload_enabled.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Enablement IMDSv2 by default
+aliases:
+  - /ec2_imdsv2_transition_payload_enabled
+---
+
+# IMDSv2 Default Behavior & Host Metadata Enhancement
+
+From 7.64.0, the Agent now enables IMDSv2 by default for enhanced security and improved metadata handling. In addition, a new field, `legacy-resolution-hostname`, has been added to the host metadata payload. This field is used to capture the AWS instance hostname using legacy resolution logic (i.e. when IMDSv2 was disabled) and is only included when needed.
+
+> **Important:** The hostname displayed by the Agent **may** change if the Agent previously did not show the instance ID as the hostname, however it will continue to use the original hostname to tag the different metrics sent by the Agent.
+
+## What’s New?
+
+1. **IMDSv2 Enabled by Default:**
+   - The Agent now defaults to using IMDSv2 for metadata resolution, increasing overall security.
+
+2. **Additional Metadata Field:**
+   - **Field Name:** `legacy-resolution-hostname`
+   - **Purpose:** Enables IMDSv2 by default and captures the AWS instance hostname using legacy (IMDSv1) logic.
```

Comment:
How does this metadata field enabled IMDSv2? Did you mean that it is included when IMDSv2 is enabled?